mmdet/datasets/objects365.py

```diff
@@ -0,0 +1,231 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os.path as osp
+
+from .api_wrappers import COCO
+from .builder import DATASETS
+from .coco import CocoDataset
+
+# images exist in annotations but not in image folder.
+objv2_ignore_list = [
```

Comment:
Is this a general problem? Or only exist in our data?

Reply:
It's a general problem